ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1078,14 +1099,20 @@ static PrjFS_Result HandleFileNotification(
     const MessageHeader* request,
     const char* relativePath,
     const char* absolutePath,
+    const char* relativeFromPath,
     bool isDirectory,
     PrjFS_NotificationType notificationType)
 {
+    // Don't send null to providers
+    relativePath = relativePath != nullptr ? relativePath : "";
+    relativeFromPath = relativeFromPath != nullptr ? relativeFromPath : "";
+    
 #ifdef DEBUG
     cout
         << "PrjFSLib.HandleFileNotification: "
-        << absolutePath
+        << (absolutePath == nullptr ? "[NULL]" : absolutePath)
         << " (root-relative: " << relativePath << ")"
```

Comment:
If we're checking that `absolutePath` is `nullptr`, do we need a similar check for `relativePath`?

Reply:
Updated so only the second path can be null